fix(compiler): remove container index from conditional instruction

crisbeto opened the pull request:
Stops passing in the `containerIndex` argument to the `conditional` instruction since it isn't being used anymore.

pkozlowski-opensource reviewed (approved):
LGTM

Reviewed-for: fw-core

crisbeto committed:
fix(compiler): remove container index from conditional instruction

Stops passing in the `containerIndex` argument to the `conditional` instruction since it isn't being used anymore.

pkozlowski-opensource commented:
This PR was merged into the repository by commit 7d5bc1c62870d9c68e06eddec229a9b8988e92ee.
